packages/fxa-settings/src/pages/InlineTotpSetup/index.stories.tsx

```diff
@@ -0,0 +1,31 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import InlineTotpSetup from '.';
+import { Meta } from '@storybook/react';
+import { MozServices } from '../../lib/types';
+import AppLayout from '../../components/AppLayout';
+import { MOCK_CODE, MOCK_EMAIL } from './mocks';
+
+export default {
+  title: 'pages/InlineTotpSetup',
+  component: InlineTotpSetup,
+} as Meta;
+
+export const Default = () => (
```

Comment:
I would love to see the second page as a story here but it requires a couple modifications. I might look at this in one of my clean up tickets this next sprint.

Reply:
Yeah! Originally I had it so that you could skip directly to the page, but it felt like it created a misleading impression as to how the component actually works so it felt like a bad idea just for the sake of storybook